src/layout/components/NavHeader/Organization.vue

```diff
@@ -99,8 +99,11 @@ export default {
     },
     currentOrgId() {
       const usingOrgIds = this.usingOrgs.map(o => o.id)
+
       let currentOrgId = this.currentOrg.id
+
       const find = usingOrgIds.indexOf(currentOrgId) > -1
+
       if (!find) {
         currentOrgId = null
       }
```

Comment:
The main issue is that your import statement contains multiple elements which is causing an error while trying to assign those values in `this.currentOrg` after calling the method `currentOrgId`, because none of them match.

Here's what should happen:

```
import currentOrg from "@org/CurrentOrg";

export default {
     ...currentOrg,
}
```